numpy/distutils/fcompiler/nag.py

```diff
@@ -63,7 +60,6 @@ class NAGFORCompiler(BaseNAGFCompiler):
         'archiver'     : ["ar", "-cr"],
         'ranlib'       : ["ranlib"]
         }
-
```

Comment:
This is a bit odd, having one blank line between methods is preferred.

Reply:
I've just put the spacing in.